src/frontend/src/app/core/state/proposal/proposal.mapper.ts

```diff
@@ -54,6 +54,57 @@ export function mapOpenProposalListResponse(
   return proposalsList;
 }
 
+export function mapClosedProposalListResponse(
+  apiResponseList: ProposalResponse[],
+): Proposal[] {
+  const proposalsList: Proposal[] = [];
```

Comment:
I'd prefer to do it like this:

```typescript
return apiResponseList.map(proposalResponse => ({
  // redacted...
}));
```

Reply:
One method should be enough for this. There will be one endpoint with a filter to return a common model.